base/doc/ltnews34.tex

```diff
@@ -176,6 +176,24 @@ \subsection{New implementation for \cs{counterwithin}}
 \emph{write appropriate description}
   
 
+
+\subsection{New default for \cs{tracinglostchars}}
+
+In 2021 the \TeX{} engines got enhanced so that \cs{tracinglostchars}
+is supporting other values than zero and one.  It now accepts the
```

Comment:
Actually the change in 2021 was only for `\tracinglostchars=3`.  `\tracinglostchars=2` was already added in e-TeX, so we don't have that as an excuse :)

Other than that, the change looks fine

Reply:
can't really argue ... description updated